receiver/gitproviderreceiver/documentation.md

```diff
@@ -34,6 +34,79 @@ Number of repositories in an organization
 | ---- | ----------- | ---------- |
 | 1 | Gauge | Int |
 
+### git.repository.pull_request.approved.time
+
+The amount of time it took a pull request to go from open to approved
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| 1 | Gauge | Int |
+
+#### Attributes
+
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| repository.name | The full name of a Git repository | Any Str |
+| branch.name | The name of a Git branch | Any Str |
+
+### git.repository.pull_request.merged.count
+
+The number of merged pull requests in a repository
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| 1 | Gauge | Int |
+
+#### Attributes
+
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| repository.name | The full name of a Git repository | Any Str |
+
+### git.repository.pull_request.merged.time
+
+The amount of time it took a pull request to go from open to merged
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| 1 | Gauge | Int |
+
+#### Attributes
+
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| repository.name | The full name of a Git repository | Any Str |
+| branch.name | The name of a Git branch | Any Str |
+
+### git.repository.pull_request.open.count
+
+The number of open pull requests in a repository
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| 1 | Gauge | Int |
+
+#### Attributes
+
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| repository.name | The full name of a Git repository | Any Str |
```

Comment:
I think we agreed to not say `the full name` and instead just say `The name of a Git repository`.